src/main.c

```diff
@@ -173,7 +223,16 @@ void main(void)
 {
     vector_init(&tests_to_run, kernel_thunk_table_size);
     vector_init(&tests_exclude, 20);
-    if (!open_output_file("D:\\kernel_tests.log")) {
+    load_name_file("D:\\name.txt");
+    char* output_file_name = "D:\\kernel_tests.log";
+    // If name_log buffer is allocated, then we know it does have actual input.
+    if (name_log) {
+        size_t name_log_length = strlen(name_log);
+        name_log_length += strlen(name_log_format) - 2; // exclude %s
+        output_file_name = calloc(name_log_length + 1, sizeof(char));
+        snprintf(output_file_name, name_log_length, name_log_format, name_log);
+    }
+    if (!open_output_file(output_file_name)) {
         return;
```

Comment:
`output_file_name` is leaking here

Reply:
App is terminated here, not an issue and does not cause memory leak into reboot dashboard. Only contiguous memory allocation can cause memory leak.